backend/library/libraries/soc-cmm-2.4.2-basic.yaml

```
@@ -0,0 +1,12536 @@
+convert_library_version: 'v2 ; Compat Mode: [0] {[v2.1] (DEFAULT) Don''t use any Compatibility
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)